sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,142 @@
+  - Removed old Ordered Queue proposal and make a note of deprecation [e8f1f29]
+
+## Bug fixes
+### SYCL Frontend and driver changes
+  - Fixed variable sharing passed by value to `parallel_for_work_group`
+    [d8ea63a]
+  - Do not produce an error if some restricted feature (e.g. exceptions) is used
+    by a function only referenced in unevaluated contexts [5eae571]
+  - Fixed problem with not cleaning up temporary files when device code split
+    feature is enabled [d86ee2f]
+  - Fixed issue with emitting wrong 'typename' keywords in the integration
+    header [c19372e]
+  - The SYCL target image registration functions have been renamed to avoid
+    conflicts with the OpenMP registration functions [82fd970]
+  - Avoid using `std::memcpy` in the device code [f39f47e]
+  - Fixed `-save-temps` option when used along with `-fsycl` [f7f4699]
+  - Also fixed link steps triggering for libraries specified using
+    `-foffload-static-lib` when no source or object file is provided [360b25b]
+
+### SYCL headers and runtime
+  - Fixed final result saturation in the host implementation of `sycl::mad_sat`
+    [54dddb4]
+  - Fixed a crash when a non-nullptr_t `0x0` value is passed to the
+    `sycl::buffer::set_final_data` method [6a0e279]
+  - Fixed an issue with copying sub-buffer between different contexts [0867a38]
+  - Resolved the problem when local accessor is a temporary object [1eed329]
+  - Fixed an issue with event not being retained when memory object is
+    constructed using interoperability constructors [0aabe7e]
+  - Fixed compilation of kernels which use `sycl::stream` for FPGA device
+    [c4dbaa2]
+  - Fixed execution graph cleanup on memory object destruction [7a75b54]
+
+## Known issues
+  - [new] Defining macro with names SUCCESS or FAILED can break SYCL headers
+    because such names are used internally
+  - [new] The format of the object files produced by the compiler can change
+    between versions. Workaround is to rebuild the application.
+  - [new] The SYCL library doesn't guarantee stable API/ABI, so applications
+    compiled with older version of the SYCL library may not work with new one.
+    Workaround is to rebuild the application.
+  - Using `cl::sycl::program` API to refer to a kernel defined in another
+    translation unit leads to undefined behaviour
+  - Linkage errors with the following message:
+    `error LNK2005: "bool const std::_Is_integral<bool>" (??$_Is_integral@_N@std@@3_NB) already defined`
+    can happen when a SYCL application is built using MS Visual Studio 2019
+    version below 16.3.0
+    For MSVC version having the error the workaround is to use -std=c++17 switch.
+
+## Prerequisites
+### Linux
+- Experimental Intel(R) CPU Runtime for OpenCL(TM) Applications with SYCL
+  support version
+  [2020.10.3.0.04](https://github.com/intel/llvm/releases/download/2020-02/oclcpuexp-2020.10.3.0.04_rel.tar.gz)
+  is recommended OpenCL CPU RT prerequisite for the SYCL compiler
```

Comment:
```suggestion
  is the recommended OpenCL CPU RT prerequisite for the DPC++ compiler
```
(the same applies to other prerequisites)

Reply:
Let's merge as is. It's end of March already. These notes are coming quite late.